tuxemon/menu/input.py

```diff
@@ -34,7 +34,7 @@ class InputMenu(Menu[InputMenuObj]):
     background = None
     draw_borders = False
 
-    def startup(
+    def __init__(
```

Comment:
Remove the `kwargs` parameter from all `__init__` methods, as it was only necessary in `startup` for preventing a LSP violation.

Reply:
We still need it for Menus.

Reply:
I have removed it for the child classes, but am leaving it here.

Reply:
Why not the explicit allowed parameters, with their actual types?

Reply:
There are 19 parameters. Maybe in a future ticket.